.evergreen/config.yml

```diff
@@ -1124,27 +1124,58 @@ axes:
   - id: os
     display_name: OS
     values:
+      # Debian
       - id: debian11
         display_name: "Debian 11"
-        run_on: debian11
+        run_on: debian11-small
       - id: debian10
         display_name: "Debian 10"
-        run_on: debian10
+        run_on: debian10-small
       - id: debian92
         display_name: "Debian 9.2"
-        run_on: debian92
-      - id: rhel70
-        display_name: "RHEL 7.0"
-        run_on: rhel70
+        run_on: debian92-small
+
+      # RHEL
+      - id: rhel90
+        display_name: "RHEL 9.0"
+        run_on: rhel90-small
+        variables:
+          SKIP_LEGACY_SHELL: "true"
+      # TODO: RHEL 8.x is not working yet
+      # - id: rhel82-arm64
+      #   display_name: "RHEL 8.2 ARM64"
+      #   run_on: rhel82-arm64-small
+      # - id: rhel80
+      #   display_name: "RHEL 8.0"
+      #   run_on: rhel80-small
+      - id: rhel76
+        display_name: "RHEL 7.6"
+        run_on: rhel76-small
       - id: rhel71-power8
         display_name: "RHEL 7.1 Power 8"
         run_on: rhel71-power8-build
       - id: rhel72-zseries
         display_name: "RHEL 7.2 zSeries"
         run_on: rhel72-zseries-build
-      - id: ubuntu1804-arm64
-        display_name: "Ubuntu 18.04 ARM64"
-        run_on: ubuntu1804-arm64-test
+
+      # Ubuntu LTS
+      - id: ubuntu2204
+        display_name: "Ubuntu 22.04 x64"
+        run_on: ubuntu2204-small
+        variables:
+          SKIP_LEGACY_SHELL: "true"
```

Comment:
Is `SKIP_LEGACY_SHELL` here in any way related to https://github.com/mongodb/mongo-php-library/pull/1035 and [PHPLIB-1028](https://jira.mongodb.org/browse/PHPLIB-1028)?

I only found `SKIP_LEGACY_SHELL` referenced in [`download-mongodb.sh`](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/2608a86a58be31670e4c7b5aef26d3db44dd3df8/.evergreen/download-mongodb.sh#L714). In that script, it is only used in a test for `-z "${SKIP_LEGACY_SHELL:-}"` (i.e. branch happens when the variable is zero-length or undefined). So in that case, defining this string as _anything_ is considered a "true" value to avoid the conditional download of the legacy shell. That seems a bit odd to me, but I suppose it's just because we tend to do explicit checks for "yes" values in our own scripts.

Why do we only do this for the Ubuntu 22.04 platforms? If the legacy shell isn't necessary at all, perhaps we should be setting `SKIP_LEGACY_SHELL` for all environments.

I see `download-mongodb.sh` is only called in our config for the `getdata` task (only executed manually for debugging). I suppose the only other place it is called is via the `bootstrap mongo-orchestration` task through `run-orchestration.sh`.